aws/resource_aws_ecs_task_definition.go

```diff
@@ -108,6 +108,44 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 							Optional: true,
 							ForceNew: true,
 						},
+
+						"docker_volume_configuration": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							ForceNew: true,
```

Comment:
Should all underlying elements be `ForceNew: true` as well? I believe in Terraform 0.11 the behavior of `ForceNew: true` propagating to all children attributes was changed/removed.